packages/libllvm/libcompiler-rt.subpackage.sh

```diff
@@ -16,22 +16,34 @@ TERMUX_SUBPKG_CONFLICTS="ndk-multilib (<< 23b-6)"
 termux_step_create_subpkg_debscripts() {
 	local RT_OPT_DIR=$TERMUX_PREFIX/opt/ndk-multilib/cross-compiler-rt
 	local RT_PATH=$TERMUX_PREFIX/lib/clang/$TERMUX_PKG_VERSION/lib/linux
+	local LLVM_MINGW_RT_OPT_DIR=$TERMUX_PREFIX/opt/llvm-mingw-w64/opt/compiler-rt
+	local LLVM_MINGW_RT_PATH=$TERMUX_PREFIX/lib/clang/$TERMUX_PKG_VERSION/lib/windows
 
 	cat <<- EOF > ./triggers
-	interest-noawait $RT_OPT_DIR
+	interest-noawait $RT_OPT_DIR $LLVM_MINGW_RT_OPT_DIR
 	EOF
 
 	cat <<- EOF > ./postinst
 	#!$TERMUX_PREFIX/bin/sh
+	if [ "\$1" = "triggered" -a "\$2" = "$LLVM_MINGW_RT_OPT_DIR" ]; then
+		ln -sf $LLVM_MINGW_RT_OPT_DIR/windows $LLVM_MINGW_RT_PATH
+		exit 0
+	fi
+
 	find $RT_PATH -type l ! -name "libclang_rt*$TERMUX_ARCH*" -exec rm -rf "{}" \;
 	if [ -e $RT_OPT_DIR ]; then
-	    find $RT_OPT_DIR -type f ! -name "libclang_rt*$TERMUX_ARCH*" -exec ln -sf "{}" $RT_PATH \;
+		find $RT_OPT_DIR -type f ! -name "libclang_rt*$TERMUX_ARCH*" -exec ln -sf "{}" $RT_PATH \;
```

Comment:
It is not necessary to use tab characters to indent in heredoc. Rather tabs would make the generated script less readable, because all leading tabs are omitted. Not critical because the content is not a Python script, but I suggest using 4 characters here.

Reply:
I mean 4 *space* characters (`    `).

Reply:
OK. Fixed now.